pandas/util/__init__.py

```diff
@@ -23,3 +23,7 @@ def __getattr__(key: str):
         return cache_readonly
 
     raise AttributeError(f"module 'pandas.util' has no attribute '{key}'")
+
+
+def capitalize_first_letter(s):
```

Comment:
because `.title` also capitalises the alias, not just the first letter:
```python
(Pdb) p self.name
'period[us]'
(Pdb) p self.name.title()
'Period[Us]'
```